vital/types/detected_object.h

```diff
@@ -121,6 +143,28 @@ class VITAL_EXPORT detected_object
    */
   void set_bounding_box( const bounding_box_d& bbox );
 
+
+  /**
+   * @brief Get geo_point from this detection.
```

Comment:
Consider "Get geodetic location ..." instead? (Also in long description; maybe there, "geodetic location (geo_point)".)